src/components/contextual/pages/pools/StakedPoolsTable.vue

```diff
@@ -0,0 +1,192 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+
+import QUERY_KEYS from '@/constants/queryKeys';
+
+import usePoolsQuery from '@/composables/queries/usePoolsQuery';
+import useWeb3 from '@/services/web3/useWeb3';
+import useUserPoolsQuery from '@/composables/queries/useUserPoolsQuery';
+
+import { FullPool } from '@/services/balancer/subgraph/types';
+import { bnum } from '@/lib/utils';
+
+import PoolsTable from '@/components/tables/PoolsTable/PoolsTable.vue';
+import StakePreview from '../../stake/StakePreview.vue';
+
+import { UserGuageSharesResponse } from './types';
+import useGraphQuery, { subgraphs } from '@/composables/queries/useGraphQuery';
+
+/** TYPES */
+type Props = {
+  userPools: FullPool[];
+};
+
+const props = withDefaults(defineProps<Props>(), {
+  userPools: () => []
+});
+
+/** STATE */
+const showStakeModal = ref(false);
+const hasStaked = ref(false);
+const stakePool = ref<FullPool | undefined>();
+
+/** QUERY ARGS */
+const userPoolsAddresses = computed(() => {
+  return props.userPools.map(pool => pool.address.toLowerCase());
+});
+
+const userStakedPools = computed(() => {
+  if (isLoadingGaugeShares.value || !gaugeSharesRes.value) return [];
+  return gaugeSharesRes.value.gaugeShares.map(share => {
+    return share.gauge.poolId;
+  });
+});
+
+/** COMPOSABLES */
+const { account } = useWeb3();
+const { data: gaugeSharesRes, isLoading: isLoadingGaugeShares } = useGraphQuery<
+  UserGuageSharesResponse
+>(subgraphs.gauge, QUERY_KEYS.Gauges.GaugeShares.User(account), () => ({
+  gaugeShares: {
+    __args: {
+      where: { user: account.value.toLowerCase() }
+    },
+    balance: 1,
+    gauge: {
+      poolId: 1
```

Comment:
Please use `true` here instead. It's confusing to read this especially with the balance attribute.